internal/service/privatelinkendpointserviceserverless/resource_privatelink_endpoint_service_serverless.go

```diff
@@ -146,6 +148,51 @@ func resourceCreate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 	return resourceRead(ctx, d, meta)
 }
 
+func resourceUpdate(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
+	connV2 := meta.(*config.MongoDBClient).AtlasV2
+	projectID := d.Get("project_id").(string)
+	instanceName := d.Get("instance_name").(string)
+	endpointID := d.Get("endpoint_id").(string)
+
+	_, _, err := connV2.ServerlessPrivateEndpointsApi.GetServerlessPrivateEndpoint(ctx, projectID, instanceName, endpointID).Execute()
+	if err != nil {
+		return diag.Errorf("error getting Serverless PrivateLink Endpoint Information: %s", err)
+	}
+
+	// only "comment" attribute update is supported, updating other attributes forces replacement of this resource
+	updateRequest := admin.ServerlessTenantEndpointUpdate{
+		Comment:      conversion.StringPtr(d.Get("comment").(string)),
+		ProviderName: d.Get("provider_name").(string),
+	}
+
+	endPoint, _, err := connV2.ServerlessPrivateEndpointsApi.UpdateServerlessPrivateEndpoint(ctx, projectID, instanceName, endpointID, &updateRequest).Execute()
+	if err != nil {
+		return diag.Errorf(ErrorServerlessServiceEndpointAdd, endpointID, err)
+	}
+
+	stateConf := &retry.StateChangeConf{
```

Comment:
is this needed when only comment is changed or the comment change in the service is synchronous and there is no need to poll the state?

Reply:
removed